WordPress/Classes/ViewRelated/Cells/TitleBadgeDisclosureCell.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  SiteTagCell.swift
+//  WordPress
+//
+//  Created by Cesar Tardaguila Moro on 2017-12-04.
+//  Copyright © 2017 WordPress. All rights reserved.
+//
+
+import UIKit
+
+final class TitleBadgeDisclosureCell: WPTableViewCell {
+    @IBOutlet weak var cellTitle: UILabel!
+    @IBOutlet weak var cellBadge: BadgeLabel!
+
+    private struct BadgeConstants {
+        static let padding: CGFloat = 6
+        static let radius: CGFloat = 9
+        static let border: CGFloat = 1
```

Comment:
Super nitpicky of me, but we like to match the format of a literal to its type.  So we'd want to format CGFloats, doubles, floats, etc. with a `.0`, if that makes sense.